src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -33,6 +33,31 @@ message NamespaceOptions {
     bool coldWritesEnabled                          = 10;
     NamespaceRuntimeOptions runtimeOptions          = 11;
     google.protobuf.BoolValue cacheBlocksOnRetrieve = 12;
+    AggregationOptions aggregationOptions           = 13;
+}
+
+message AggregationOptions {
+    repeated Aggregation aggregations = 1;
+}
+
+message Aggregation {
+    AggregationAttributes attributes    = 1;
```

Comment:
why not just flatten the fields on this message?

Reply:
Yeah, we could. Mostly just mirroring the structure this data will ultimately make its way into: https://github.com/m3db/m3/blob/27c7ce231a0a748627495d1940d94b49b86c5c40/src/query/storage/m3/cluster.go#L78